pkg/scheduler/framework/session_plugins.go

```diff
@@ -665,6 +688,110 @@ func (ssn *Session) PredicateFn(task *api.TaskInfo, node *api.NodeInfo) error {
 	return nil
 }
 
+// InitCycleStateFn 调用插件的初始化CycleState函数
+func (ssn *Session) InitCycleStateFn(ctx context.Context, state *k8sframework.CycleState) error {
+	for _, tier := range ssn.Tiers {
+		for _, plugin := range tier.Plugins {
+			if !isEnabled(plugin.EnabledAllocatable) && !isEnabled(plugin.EnabledPreemptable) {
+				continue
+			}
+			pfn, found := ssn.initCycleStateFns[plugin.Name]
+			if !found {
+				continue
+			}
+			klog.Infof("InitCycleStateFn, plugin: %v", plugin.Name)
+			err := pfn(ctx, state)
+			if err != nil {
+				return err
+			}
+		}
+	}
+	return nil
+}
+
+// ConcurrentAllocatableFn 调用插件的并发分配函数
+func (ssn *Session) ConcurrentAllocatableFn(ctx context.Context, state *k8sframework.CycleState, queue *api.QueueInfo, task *api.TaskInfo) bool {
```

Comment:
Where does `ConcurrentAllocatableFn` reflect `Concurrent`? I didn't see it. And the name will be easy to confuse users, there are both `AllocatableFn` and `ConcurrentAllocatableFn`